### PR TITLE
[ORCA-642] Add getLPSupply function

### PR DESCRIPTION
**Pull request by odcheung**
- Add the addLPSupply function

[Tests]
    - Verified LP Supply token count equals spl-token supply

### CODE REVIEW

File: src/model/orca-pool.ts [hunk 1]
```diff
@@ -43,5 +43,11 @@ class OrcaPoolImpl implements OrcaPool {
         }
 
         return shiftByDecimal(result.amount, this.poolParams.poolTokenDecimals).toNumber()
-    };
+    }
+
+    public async getLPSupply(): Promise<number> {
+        const context = await this.connection.getTokenSupply(this.poolParams.poolTokenMint)
+        const amt = new u64(context.value.amount)
+        return shiftByDecimal(amt, this.poolParams.poolTokenDecimals).toNumber()
```
Comment (rawfalafel) [inline src/model/orca-pool.ts:51]: Nitpick, but looks like this repo could use a linter to automatically add semicolons and stuff

**Review by rawfalafel — APPROVED** [review]
looks good :+1:

File: src/model/orca-pool.ts (hunk 1 above)
Comment (odcheung) [reply src/model/orca-pool.ts:51]: good point. let me consolidate our eslint so we can be consistent (I killed the semi-colon because I'm used to swift)

Comment (rawfalafel) [reply src/model/orca-pool.ts:51]: Ohhh, haha :rofl:

**Pull request merged**